en/middleware/micrortps.md

```diff
@@ -35,7 +35,9 @@ The *Client* application is also compiled and built into the firmware as part of
 
 ## Supported uORB messages
 
-The set of uORB topics that can be supported by the bridge are listed in the **msg/CMakeLists.txt**. You can modify the list to change what uORB messages can be published/subscribed using RTPS in the **.cmake** file (**cmake/configs**) for each target platform.
```

Comment:
@VicenteEprosima I modified this because **msg/CMakeLists.txt** does not include the set of topics - those are in the cmake files. 
From what I can see the only people who need to know about CMakeLists.txt are those that are changing  our support for RTPS (so for example, this might be where we would define a default list of uORB topics for all targets). Do you think we need to still say something about what this file is for?